### PR TITLE
Feature/show dev cards

### PR DESCRIPTION
**Pull request by sumnerfit**
When clicking a player, you can now see how many resource cards (hidden), dev cards(hidden), played knights (shown) and longest_road / largest army (shown).

Clicking Knight shows knight card dialog.

Test with: {url}/?startWithCards=5

- buy dev cards and make a longest road.
- check in other player tab that cards are shown.. but hidden where appropriate.
- check that Victory Points from cards are not displayed to other players (Thanks Tim)
- Fixed "Card Rules" modal
- added @media for us small screeners to have the right panel fit in full screen mode.... shouldn't effect larger screens.

### CODE REVIEW

**Comment by Luke-Nukem** [comment]
@sumnerfit looks like there's some minor conflicts with master. I would resolve them myself but I'm unsure if I should.

Also curious as to whether `turn_complete: false,` is still used anywhere in the turn update.

**Comment by sumnerfit** [comment]
Rebased PR and added a dice roll fix to stop fixed dice in normal game play

**Comment by Luke-Nukem** [comment]
Looks great so far! Though there is a small issue:
screenshot from 2017-10-07 17-00-31

File: public/js/game/client.js [hunk 1]
```diff
@@ -32,9 +32,7 @@ $(document)
     $(".btn-control-maximize").click(function(){
       $(".btn-control-maximize").removeClass('btn-msg');
       $(".btn-control-maximize").toggleClass('btn-plus');
-      $(".score").toggleClass("score--back");
-      $(".game_chat").toggleClass("game_chat--back");
-      //$(".score").slideToggle();
+      $(".score").slideToggle();
```
Comment (Luke-Nukem) [inline public/js/game/client.js]: Needs to be reverted.

File: public/js/game/client.js [hunk 2]
```diff
@@ -32,9 +32,7 @@ $(document)
     $(".btn-control-maximize").click(function(){
       $(".btn-control-maximize").removeClass('btn-msg');
       $(".btn-control-maximize").toggleClass('btn-plus');
-      $(".score").toggleClass("score--back");
-      $(".game_chat").toggleClass("game_chat--back");
-      //$(".score").slideToggle();
```
Comment (Luke-Nukem) [inline public/js/game/client.js]: This and line above shouldn't have been removed - may be part of a bad merge?

File: public/js/game/client.js [hunk 3]
```diff
@@ -74,7 +72,6 @@ $(document)
       var popupData = [
         ["show_message", "block"],
         ["player_count", data.player_count],
-        ["player_list", data.player_list],
```
Comment (Luke-Nukem) [inline public/js/game/client.js]: Shouldn't have been removed.

File: public/js/game/client.js [hunk 4]
```diff
@@ -197,7 +194,6 @@ $(document)
         build_popup_setup_complete();
 
       } else if (data.data_type === 'setup_phase') {
-        $(".game_chat").removeClass('game_chat_top_z');
```
Comment (Luke-Nukem) [inline public/js/game/client.js]: Was this and line 227 removed on purpose?

**Comment by Luke-Nukem** [comment]
The easiest way for you to double check everything here will be to:
1. `git rebase -i a787d9543a979dd70a8eff03ec601a6995149022^` and in the editor change each word next to your commits from *pick* to *edit*
2. then `git reset HEAD^` to unstage those changes, run a `git diff master` to see what has changed, then make any corrections and test
3. `git add` those changes/unstaged files and then `git commit --amend`
4. finally, `git rebase --continue` to proceed to the next commit to change (and back to step 2)

**Comment by sumnerfit** [comment]
I ran the commands as above and found two instances of client.js having portions removed (in areas I wasn't working). 

We can try it again now.

**Comment by Luke-Nukem** [comment]
Merging. Looks fine. Works fine.

**Pull request merged**